svgsynoptic2/taurussynopticwidget.py

```diff
@@ -151,6 +154,19 @@ def filter_fragment(self, model, value):
                 pass
         return value
 
+    def set_custom_value(self, model, attr_value):
+        # this is the special case where value is modified for PyAlarm class
+        # when alarm is trigered PyAlarm attribute returns 'True' value, icon on synoptic is green
+        # set_custom_value change value for better alarms visualisation
+        # function can be customised for any other tango device classes
+        try:
+            if 'PyAlarm' in PyTango.get_device_proxy(model).info().dev_class:
+                return not(attr_value)
+        except Exception as e:
+            print('Problem to set custom value {0} \n for model {1}'.format(e, model))
+        finally:
+            return attr_value
```

Comment:
Using `finally` like this cause that there is always returned `attr_value`, no matter what happens in `try`. There should be return after `except` (not inside).